qa/integration-tests/src/test/java/io/camunda/it/utils/ZeebeClientTestFactory.java

```diff
@@ -31,6 +28,9 @@
 
 public final class ZeebeClientTestFactory implements AutoCloseable {
 
+  private static final String DEFAULT_USER_USERNAME = "demo";
+  private static final String DEFAULT_USER_PASSWORD = "demo";
```

Comment:
❓ Can we move this variables in the configuration class, and use them everywhere ? Not sure if it make sense